JSON_Bourne/get_webpage.py

```diff
@@ -0,0 +1,112 @@
+from lxml import html
+import requests
+import json
+
+
+def get_info(url):
+    page = requests.get(url)
+    tree = html.fromstring(page.content)
+
+    titles = tree.xpath("//tr/th/a")
+    titles = [t.text for t in titles]
+
+    status = tree.xpath("//table[2]/tbody/tr/td[1]")
+    # status = [s.text for s in status]
+    status_text = list()
+    for s in status:
+        if s.text is None:
+            # This means there is an extra <font> in the XML to change the colour of the value
+            # This only happens when the PV is "Disconnected"
+            # This assumption speeds up the program significantly
+            status_text.append("Disconnected")
+        else:
+            status_text.append(s.text)
+
+    info = tree.xpath("//table[2]/tbody/tr/td[3]")
+    values = list()
+    alarms = list()
+
+    for i in range(len(titles)):
+        if "STARTTIME" in titles[i]:
+            # Time is delimited by \t
+            if info[i].text == "null":
+                values.append("null")
+                alarms.append("null")
+            else:
+                single = info[i].text.split("\t", 2)
```

Comment:
Single is an odd variable name. It doesn't really describe what is going on here